src/main/java/core/basesyntax/StringStream.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.PrintStream;
+
+public class StringStream extends PrintStream {
+    private String printString = "";
+
+    public StringStream() throws FileNotFoundException {
+        super(new FileOutputStream("file"));
```

Comment:
Let's wrap code that creates new FileOutputStream into try catch. Catch exception and rethrow RuntimeException. Than you won't have to handle all these useless FileNotFoundExceptions.

Reply:
done